picky/Cargo.toml

```diff
@@ -46,6 +47,12 @@ reqwest = { version = "0.11.14", default-features = false, features = ["blocking
 rand = "0.8.5"
 num-bigint-dig = "0.8.2"
 
+ed25519-dalek = "1"
+# NOTE: x25519-dalek 2.0 is still in rc, so we use 1.0 for now.
+x25519-dalek = "1"
+# ed25519 and x25519 use older versions of `rand` crate
+rand_ed = { package = "rand", version = "0.7" }
```

Comment:
This kinda sucks (and also `ed25519-dalek/x25519-dalek` do not reexport `rand` so we should do it manually) but we must live with 2 rand crates...

When version 2.0 for `ed25519-dalek/x25519-dalek` will be released, this will be removed

Reply:
I understand where you come from. Let’s roll with this for now.